sdks/python/setup.py

```diff
@@ -546,20 +556,14 @@ def get_portability_package_data():
           # in https://github.com/apache/beam/blob/master/sdks/python/tox.ini
           # For more info, see
           # https://docs.google.com/document/d/1c84Gc-cZRCfrU8f7kWGsNR2o8oSRjCM-dGHO9KvPWPw/edit?usp=sharing
-          'torch': [
-              'torch<=1.13.0,<=2.0.0'
-          ],
-          'tensorflow': [
-              'tensorflow>=2.12rc1,<2.13'
-          ],
+          'torch': ['torch<=1.13.0,<=2.0.0'],
```

Comment:
we should use strict upper bound, so <2.1.0 or <2.0

Reply:
this was just a formatting change, the torch bound has been in that form

Reply:
i know, my ask was to fix it while at it, since it's a bad practice to have them: https://cwiki.apache.org/confluence/display/BEAM/Dependency+management+guidelines+for+Beam+Python+SDK+maintainers#DependencymanagementguidelinesforBeamPythonSDKmaintainers-Howtoaddanewdependency?

My bad, I wasn't clear.